.gitignore

```diff
@@ -54,4 +54,10 @@ bower.json
 environment.sh
 
 # Ignore database.yml from version control...
-config/database.yml
\ No newline at end of file
+config/database.yml
+
+config/initializers/s3.rb
+config/sitemap.rb
+
+config/initializers/recaptcha.rb
+db/structure.sql
```

Comment:
We don't want to ignore structure.sql, as we are currently using this in place of a schema.rb file.

Reply:
Got it!